### PR TITLE
[Signup fields III] Profile form

### PR DESCRIPTION
**Pull request by gonzaloriestra**
Related to CartoDB/product#337

Added the new fields to the user profile and enabled synchronization with central.

:warning:  **DO NOT MERGE UNTIL https://github.com/CartoDB/cartodb/pull/14963 IS DEPLOYED**

### CODE REVIEW

**Comment by gonzaloriestra** [comment]
retest this please

File: spec/services/carto/user_metadata_export_service_spec.rb [hunk 1]
```diff
@@ -144,9 +144,12 @@ def destroy_user(user = @user)
     end
 
     it 'includes all user model attributes' do
+      expected_attrs = @user.attributes.symbolize_keys.keys - [:rate_limit_id, :company_employees, :use_case] +
```
Comment (alrocar) [inline spec/services/carto/user_metadata_export_service_spec.rb]: let's remove `:company_employees, :use_case` from here as well

Comment (gonzaloriestra) [reply spec/services/carto/user_metadata_export_service_spec.rb]: Done!

**Comment by gonzaloriestra** [comment]
retest this please

**Comment by oleurud** [comment]
Acceptance :green_heart:  :question: 

Everything seems to be ok, in cartodb, central and dbs, but in all cases, I have received a 504 error (gateway timeout) by nginx in the account creation. I suppose, this is happening in master too and it is not a due to these changes.

**Comment by oleurud** [comment]
It is happening with `master` branchs too. So I would say:

Acceptance :green_heart:

**Comment by alejandraarri** [comment]
Retest this please

**Comment by gonzaloriestra** [comment]
retest this please

**Comment by gonzaloriestra** [comment]
retest this please

**Comment by gonzaloriestra** [comment]
retest this please

**Pull request merged**